README.md

```diff
@@ -136,7 +136,7 @@ Note that if CPU limits are set too low, kube-state-metrics' internal queues wil
 
 ### kube-state-metrics vs. Heapster
```

Comment:
The title needs to be changed from "kube-state-metrics vs. Heapster" to "kube-state-metrics vs. metrics-server", and all the following description of the heapster also needs to be modified.

Reply:
As commented in https://github.com/kubernetes/kube-state-metrics/issues/518#issuecomment-414117618, i suggest we add metrics-server instead of replace it.